iet.so/include/action.h

```diff
@@ -118,8 +118,8 @@ class iet_action: public rvs::actionbase {
     //! list of GPUs (along with some identification data) which are
     //! selected for EDPp test
     std::vector<gpu_hwmon_info> edpp_gpus;
-
-
+    std::map<int, int> hip_to_smi_idxs;
```

Comment:
I am not sure of the norms followed in RVS still,
for , hip_to_smi_idxs;
we can follow Hungarian notation of 'm'  to indicat its member of that class 
( never confuse type with kind in Hungarian : https://www.joelonsoftware.com/2005/05/11/making-wrong-code-look-wrong/)

Reply:
agreed, but kept naming convention consistent with the existing fields